app/uk/gov/hmrc/crsfatcafimanagement/models/CADXRequestModels/RequestCommon.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.crsfatcafimanagement.models.CADXRequestModels
+
+import play.api.libs.json.JsonConfiguration.Aux
+import play.api.libs.json.{Json, JsonConfiguration, JsonNaming, OFormat}
+import uk.gov.hmrc.crsfatcafimanagement.models.RequestType
+
+final case class RequestCommon(
+  originatingSystem: String,
+  transmittingSystem: String,
+  requestType: RequestType,
+  regime: String,
+  requestParameters: List[RequestParameter]
+)
+
+object RequestCommon {
+  implicit val jsonConfig: Aux[Json.MacroOptions] = JsonConfiguration(naming = JsonNaming.PascalCase)
```

Comment:
We have introduced this `JsonNaming.PascalCase` for some, but not for all case classes. Shouldn't it be used for all models inside `CADXRequestModels` ?
If we should use for all, we can also introduce package object and provide once and it gets applied to all?

```
package uk.gov.hmrc.crsfatcafimanagement.models.CADXRequestModels

import play.api.libs.json.{Json, OFormat}

final case class RequestParameter(paramName: String, paramValue: String)

object RequestParameter {
  implicit val format: OFormat[RequestParameter] = Json.format[RequestParameter]
}
```

Also, I believe there should be a test which validates that this `PascalCase` conversion is in fact working, like a unit test to and from `json`?

Reply:
[following discussion with Omer]
Happy to leave as is for now because the dummy space isn't consistent in its case conventions anyway - if it works out neater in future the above solution would be neat 👍